application.py

```diff
@@ -3,7 +3,25 @@
 import os
 import re
 from app import create_app
-from flask.ext.script import Manager, Server
+from flask.ext.script import Manager, Server as _Server
+
+
+def get_extra_files():
+    def extra_files():
+        for dirname, dirs, files in os.walk('./app/content'):
+            for filename in files:
+                filename = os.path.join(dirname, filename)
+                if os.path.isfile(filename):
+                    print(filename)
+                    yield filename
+    return extra_files()
```

Comment:
My still-hacky understanding of how generators work. Squashed it out in 63edd74.